business_requirement_deliverable_cost/models/business.py

```diff
@@ -49,7 +49,7 @@ def _get_partner(self):
         br_id = br_deliverable = False
         if self.business_requirement_deliverable_id.id:
             br_deliverable = self.business_requirement_deliverable_id
-        if br_deliverable.business_requirement_id.id:
+        if br_deliverable and br_deliverable.business_requirement_id.id:
```

Comment:
probably much better, but seriously I couldn't understand 100% how it works Coveralls since I only see a diff on code as an explanation of decreasing coverall so thats why I back to the oldest code. I will commit again with the improvement you marked and let's see if coveralls remains the same.